dev/io.openliberty.mpTelemetry.1.0.thirdparty/bnd.bnd

```diff
@@ -11,19 +11,16 @@
 -include= ~../cnf/resources/bnd/bundle.props,~../cnf/resources/bnd/transform.props
 bVersion=1.0
 publish.wlp.jar.suffix: dev/api/third-party
-openTelemetryVersion=1.19.0
-alphaOpenTelemetryVersion=1.19.0.alpha
 
 Bundle-SymbolicName: io.openliberty.mpTelemetry.1.0.thirdparty; singleton:=true
 
 javac.source: 11
 javac.target: 11
 
-src: src
+openTelemetryVersion=1.19.0
+alphaOpenTelemetryVersion=1.19.0.alpha
 
-Export-Package: \
-  io.opentelemetry.api; version=${openTelemetryVersion},\
-  io.opentelemetry.api.baggage; version=${openTelemetryVersion},\
+Private-Package: \
   io.opentelemetry.api.common; version=${openTelemetryVersion},\
```

Comment:
What happened to these two packages. Shouldn't they still be included?

Reply:
I have added them back in now